shared/bundle_analysis/parsers/v2.py

```diff
@@ -163,7 +163,7 @@ def parse(self, path: str) -> Tuple[int, str]:
     def _asset_type(self, name: str) -> AssetType:
         extension = get_extension(name)
 
-        if extension in ["js"]:
+        if extension in ["js", "mjs", "cjs"]:
```

Comment:
btw there is also the v1 parser, does this change mean only v2 of the plugin schema can support mjs and cjs?